scripts/install.sh

```diff
@@ -11,6 +11,7 @@ KEEP_CORE_PATH=$PWD
 KEEP_BEACON_SOL_PATH="$KEEP_CORE_PATH/solidity/random-beacon"
 KEEP_ECDSA_SOL_PATH="$KEEP_CORE_PATH/solidity/ecdsa"
 TMP="$KEEP_CORE_PATH/tmp"
+OPENZEPPELIN_MANIFEST=".openzeppelin"
```

Comment:
Maybe we could remove just the file for the current network?

```suggestion
OPENZEPPELIN_MANIFEST=".openzeppelin/$NETWORK.json"
```

Reply:
$NETWORK is "development", but the file that is created on my local is called `unknown-1101.json` so it won't match. I'll see if there's a config param somewhere where we can specify a manifest file name. I'd rather not parse chain id in bash.

Reply:
I think the easiest way of removing that specific file is to take advantage of the chain id which is hardcoded in other config files for the development env. [2ea98e3](https://github.com/keep-network/keep-core/pull/3131/commits/2ea98e3c019ef9689b6a3deb494254a0702f5a78)